PANCHANGE_data_preprocessing2020-07-15.Rmd

```diff
@@ -374,15 +394,9 @@ glad.gad.raw.id <- glad.gad.raw %>%
     gad.trouble_relaxing_prepan = gad.trouble_relaxing,
     gad.sit_restless_hard_prepan = gad.sit_restless_hard,
     gad.becoming_easily_annoyed_or_irritable_prepan = gad.becoming_easily_annoyed_or_irritable,
-    gad.awful_feeling_afraid_happen_prepan = gad.awful_feeling_afraid_happen,
-    gad.feeling_nervous_anxious_or_on_edge_prepan_numeric = gad.feeling_nervous_anxious_or_on_edge_numeric,
-    gad.control_worrying_stop_prepan_numeric = gad.control_worrying_stop_numeric,
-    gad.worrying_too_much_about_different_things_prepan_numeric = gad.worrying_too_much_about_different_things_numeric,
-    gad.trouble_relaxing_prepan_numeric = gad.trouble_relaxing_numeric,
-    gad.sit_restless_hard_prepan_numeric = gad.sit_restless_hard_numeric,
-    gad.becoming_easily_annoyed_or_irritable_prepan_numeric = gad.becoming_easily_annoyed_or_irritable_numeric,
-    gad.awful_feeling_afraid_happen_prepan_numeric = gad.awful_feeling_afraid_happen_numeric
```

Comment:
Why have these been removed but not added back in?

Reply:
I realise now that these are the numeric ones! Ignore the above comment!

Reply:
but wait.. there is one line too many removed